src/test/java/redis/clients/jedis/doc/SetGetExample.java

```diff
@@ -0,0 +1,35 @@
+//EXAMPLE: set_and_get
+//HIDE_START
+package redis.clients.jedis.doc;
```

Comment:
Please add a file to define this package. Something like https://github.com/redis/jedis/blob/master/src/main/java/redis/clients/jedis/args/package-info.java

Reply:
Will do.

Reply:
@dmaier-redislabs I've added one. Feel free add comments, instructions there.